### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by aeosynth** [tbranyen/backbone-boilerplate#137]
https://github.com/tbranyen/backbone-boilerplate/issues/91#issuecomment-7724460

> It's a wiki feel free to contribute in places you can.

https://github.com/tbranyen/backbone-boilerplate/wiki/_history

> tbranyen 11 hours ago:  Revert 3096a9ce05ebe9c4b877de33baf8d00b460fa711 ... 2227936283784b7590a9d1028f24bec957f546b8 on Running examples [4657e27]

I'm getting mixed messages here. My changes looked good to me - I removed non-working examples and (IMO) simplified the text. I guess that page is not a place I can contribute? I also deleted a bunch of blank pages, and refactored the [Installation](https://github.com/tbranyen/backbone-boilerplate/wiki/Installation) page, but maybe those too are places where I can't actually contribute, and all my changes should be reverted.

**tl;dr** do you want help on the wiki or not?
